### ISSUE TITLE
JS module requires an absolute path to serve

### ISSUE THREAD

**Issue by timwis** [yoshuawuyts/bankai#46]
It appears `bankai.js(browserify, './src/index.js')` throws an error `Cannot find module './src/index.js'`. It only works when I give it the full path, `/Users/tim/...etc`. I think this is because of `require.resolve` on [this line](https://github.com/yoshuawuyts/bankai/blob/master/handler-js.js#L29). Not very familiar with `resolve` but would `path.resolve` be a better fit perhaps? Or am I just using it incorrectly?

**Comment by yoshuawuyts** [comment]
Hmh yeah could be - should dig further into this

**Comment by timwis** [comment]
For now I use `__dirname + '/src...'`

**Comment by marionebl** [comment]
When #59 lands `bankai` consistently resolves from `cwd` by default. To match your intended usage you could (matching browserify):

```js
bankai()(browserify, './src/index.js', {basedir: __dirname})
```

`__dirname + './src/index.js'` still works, though.

**Comment by marionebl** [comment]
Fixed with [v3.3.0](https://github.com/yoshuawuyts/bankai/releases/tag/v3.3.0)
